docs/swarm/index.md

```diff
@@ -33,44 +33,45 @@ services. You don't need additional orchestration software to create or manage
 a Swarm.
 
 * **Decentralized design:** Instead of handling differentiation between node
-roles at deployment time, Swarm handles any specialization at runtime. You can
-deploy both kinds of nodes, managers and workers, using the Docker Engine.
-This means you can build an entire Swarm from a single disk image.
+roles at deployment time, the Docker Engine handles any specialization at
+runtime. You can deploy both kinds of nodes, managers and workers, using the
+Docker Engine. This means you can build an entire Swarm from a single disk
+image.
 
-* **Declarative service model:** Swarm uses a declarative syntax to let you
-define the desired state of the various services in your application stack.
-For example, you might describe an application comprised of a web front end
-service with message queueing services and a database backend.
+* **Declarative service model:** Docker Engine uses a declarative syntax to let
+you define the desired state of the various services in your application
+stack. For example, you might describe an application comprised of a web front
+end service with message queueing services and a database backend.
 
-* **Desired state reconciliation:** Swarm constantly monitors the cluster state
-and reconciles any differences between the actual state your expressed desired
-state.
+* **Desired state reconciliation:** The Swarm manager node constantly monitors
+the cluster state and reconcile any differences between the actual state your
+expressed desired state.
 
 * **Multi-host networking:** You can specify an overlay network for your
-application. Swarm automatically assigns addresses to the containers on the
-overlay network when it initializes or updates the application.
+services. The Swarm manager automatically assigns addresses to the containers
+on the overlay network when it initializes or updates the application.
 
-* **Service discovery:** Swarm assigns each service a unique DNS name and load
-balances running containers. Each Swarm has an internal DNS server that can
-query every container in the cluster using DNS.
+* **Service discovery:** The Swarm manager assigns each service a unique DNS
+name and load balances running containers. Each Swarm has an internal DNS
+server that can query every container in the cluster using DNS.
```

Comment:
"Each Swarm has an internal DNS server that can query every container in the cluster using DNS." -> "Every container   can be queried thru a DNS server embedded in Swarm."?